package-system/mcpp/.gitattributes

```diff
@@ -1 +1,2 @@
-mcpp-2.7.2-az.2.patch text eol=lf
\ No newline at end of file
+mcpp-2.7.2-az.2.patch text eol=lf
+mcpp_2.7.2-az.3.patch text eol=lf
```

Comment:
please add a newline at the end of this file.

Reply:
done.